keras_cv/metrics/coco/numerical_tests/mean_average_precision_test.py

```diff
@@ -21,19 +21,22 @@
 
 SAMPLE_FILE = os.path.dirname(os.path.abspath(__file__)) + "/sample_boxes.npz"
 
+delta = 0.05
+
```

Comment:
Here, instead probably we cannot go lower until the area calc is so different from the reference implementation.
It could be nice to have a low-level test only on the area.

Reply:
Yeah, this is about as good as it is going to get most likely unless someone discovers another discrepancy.